hail/python/hail/methods/impex.py

```diff
@@ -651,8 +642,8 @@ def import_bed(path, reference_genome='default', skip_invalid_intervals=False, *
 
     Warning
     -------
-    UCSC BED files are 0-indexed and end-exclusive. The line "5  100  105"
-    will contain locus ``5:105`` but not ``5:100``. Details
+    UCSC BED files are 0-indexed, exclusive of the start, and inclusive of the end.
+    The line "5  100  105" will contain locus ``5:105`` but not ``5:100``. Details
```

Comment:
This isn't quite right. I think I would say
```
Intervals in UCSC BED files are 0-indexed and half open.
The line "5  100  105" corresponds to the interval ``[5:101-5:106)``
in Hail's 1-indexed notation.
```
with the link to go into details.